aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -482,6 +482,17 @@ In previous versions of ASP.NET Core, calling <xref:Microsoft.AspNetCore.Builder
 
 This scenario is fixed in ASP.NET Core 3.0. The host enables the [Forwarded Headers Middleware](xref:host-and-deploy/proxy-load-balancer#forwarded-headers-middleware-options) when the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is set to `true`. `ASPNETCORE_FORWARDEDHEADERS_ENABLED` is set to `true` in our container images.
 
+## Performance improvements
+
+Throughout the framework, ASP.NET Core 3.0 includes many improvements that reduce memory allocations and improve throughput:
```

Comment:
ASP.NET Core 3.0 includes many improvements that reduce memory usage and improve throughput:

Reply:
I left the opener for emphasis on the work.